config/locales/users/en.yml

```diff
@@ -36,3 +36,5 @@ en:
         </ul>
       overview_html: We’ve updated our %{link}. Please review and check the box below
         to continue.
+    phones:
```

Comment:
Looks like these are out of order and can be corrected by running `make normalize_yaml`.